exporters/prometheus/test/prometheus_test_helper.h

```diff
@@ -0,0 +1,237 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include "opentelemetry/version.h"
+
+namespace metric_sdk      = opentelemetry::sdk::metrics;
+namespace nostd           = opentelemetry::nostd;
+namespace exportermetrics = opentelemetry::exporter::metrics;
+
+/**
+ * Helper function to create ResourceMetrics
+ */
+metric_sdk::ResourceMetrics CreateSumPointData()
+{
+  metric_sdk::SumPointData sum_point_data{};
+  sum_point_data.value_ = 10.0;
+  metric_sdk::SumPointData sum_point_data2{};
+  sum_point_data2.value_ = 20.0;
+  metric_sdk::ResourceMetrics data;
+  auto resource = opentelemetry::sdk::resource::Resource::Create(
+      opentelemetry::sdk::resource::ResourceAttributes{});
+  data.resource_ = &resource;
+  auto instrumentation_library =
+      opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary::Create("library_name",
+                                                                                 "1.2.0");
+  metric_sdk::MetricData metric_data{
+      metric_sdk::InstrumentDescriptor{"library_name", "description", "unit",
+                                       metric_sdk::InstrumentType::kCounter,
+                                       metric_sdk::InstrumentValueType::kDouble},
+      opentelemetry::common::SystemTimestamp{}, opentelemetry::common::SystemTimestamp{},
+      std::vector<metric_sdk::PointDataAttributes>{
+          {metric_sdk::PointAttributes{}, sum_point_data},
+          {metric_sdk::PointAttributes{}, sum_point_data2}}};
```

Comment:
nit - in real world scenario, we won't have two metric data with same set of (even empty) attribute values, as they would get aggregated(added) while generating metrics data. Would be good use-case to have different set of attributes.

Reply:
added different attributes.